frontend/src/modules/Dashboard/Components/Groups.tsx

```diff
@@ -0,0 +1,65 @@
+// export { }
+import React from 'react'
+
+import {
+  StyledContainer,
+  StyledGroupArea,
+  StyledGroupCardArea,
+  StyledTextBox,
+  StyledSmallText,
+  StyledClassesContainer,
+  StyledNoClasses,
+} from 'Dashboard/Styles/Groups.style'
+
+import { GroupCard } from 'Dashboard/Components/GroupCard'
+import { CourseInfo } from 'Dashboard/Types'
+
+export const Groups = ({ groups }: ModalProps & GroupsProps) => {
+  return (
+    <StyledContainer>
+      {groups.length === 0 && (
+        <React.Fragment>
+          <StyledTextBox>
+            <h2>No Classes to Show</h2>
+          </StyledTextBox>
+          <StyledSmallText>
+            Once students request study partners, they'll automatically be
+            placed into classes on this page!
+          </StyledSmallText>
+          <StyledClassesContainer>
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+            <StyledNoClasses />
+          </StyledClassesContainer>
```

Comment:
There's too much repetition here. I feel like you should just loop this 8 times to clean it up.

Reply:
Also I think it's worth noting that this doesn't preserve how many there are in each row, and I saw on the designs there are 4 of them in each row. Iteration through some list would also solve this issue (to have a fixed number of elements on each row). I'm not sure if this is intended behavior though, so if it is, ignore this comment.